packages/medusa/src/interfaces/price-selection-strategy.ts

```diff
@@ -56,6 +57,8 @@ export type PriceSelectionContext = {
   region_id?: string
   currency_code?: string
   include_discount_prices?: boolean
+  region?: Region
+  variant?: ProductVariant
```

Comment:
An alternative approach would be to make the `get*Price` functions in the PricingService work with bulk inputs so fetching can be done once instead of having to referch the same data multiple times

Reply:
Yes the bulk is the other option i am looking at like i ve done for the generate and other methods. i completely agree on just passing the data instead of the entire entity btw. I usually do that but ive forgot to change it after all